src/main/java/mods/railcraft/common/plugins/forge/PlayerPlugin.java

```diff
@@ -61,7 +91,7 @@ public static String getUsername(World world, GameProfile gameProfile) {
         String username = gameProfile.getName();
         if (username != null && !username.equals(""))
             return username;
-        return "[Unknown]";
+        return "[Railcraft]";
```

Comment:
Also... "[Railcraft]" should generally have similar to admin permissions, whereas "[Unknown]" would be fully restricted. So returning "[Railcraft]" as a fallback is not generally safe.